applications/ShapeOptimizationApplication/tests/algorithm_bead_optimization_test/run_test.py

```diff
@@ -51,10 +51,3 @@
     TestCase().assertAlmostEqual(resulting_penalty_factor, 3.26430E+01,5)
 
 os.chdir(original_directory)
-
-# Cleaning
```

Comment:
See my comment above. Why did you remove all the lines? Without writing the design surface we still need the first two lines, right?

There are more files following where this comment applies.